Library/Homebrew/messages.rb

```diff
@@ -0,0 +1,36 @@
+# A Messages object collects messages that may need to be displayed together
+# at the end of a multi-step `brew` command run
+class Messages
```

Comment:
Could we get some basic unit tests for this class?

Reply:
Sure. Give me a little while...

Reply:
Do you want to see the `Messages` class tested in isolation in-process, or an example of shelling out to a `brew` command that produces collected caveats?

Reply:
@apjanke Tested in process/unit tests please, thanks!

Reply:
@apjanke Gentle ping on some unit tests, thanks!

Reply:
Added unit tests.